pandas/io/sas/sas7bdat.py

```diff
@@ -169,7 +169,7 @@ def _get_properties(self):
         if buf in const.encoding_names:
             self.file_encoding = const.encoding_names[buf]
         else:
-            self.file_encoding = "unknown (code={name!s})".format(name=buf)
+            self.file_encoding = f"unknown (code={str(buf)})"
```

Comment:
```suggestion
            self.file_encoding = f"unknown (code={buf})"
```

Reply:
I think we've been cleaning these up elsewhere; shouldn't need the `str()` call

Reply:
changed, thanks